test/plugins/test_ftintitle.py

```diff
@@ -175,31 +175,20 @@ def test_split_on_feat(self):
         parts = ftintitle.split_on_feat("Alice defeat Bob")
         assert parts == ("Alice defeat Bob", None)
 
-    def test_contains_feat_artist(self):
+    def test_contains_feat(self):
         assert ftintitle.contains_feat("Alice ft. Bob")
         assert ftintitle.contains_feat("Alice feat. Bob")
         assert ftintitle.contains_feat("Alice feat Bob")
         assert ftintitle.contains_feat("Alice featuring Bob")
-        assert ftintitle.contains_feat("Alice & Bob")
-        assert ftintitle.contains_feat("Alice and Bob")
-        assert ftintitle.contains_feat("Alice With Bob")
         assert ftintitle.contains_feat("Alice (ft. Bob)")
         assert ftintitle.contains_feat("Alice (feat. Bob)")
         assert ftintitle.contains_feat("Alice [ft. Bob]")
         assert ftintitle.contains_feat("Alice [feat. Bob]")
         assert not ftintitle.contains_feat("Alice defeat Bob")
         assert not ftintitle.contains_feat("Aliceft.Bob")
         assert not ftintitle.contains_feat("Alice (defeat Bob)")
-
-    def test_contains_feat_title(self):
-        assert ftintitle.contains_feat(
-            "Live and Let Go (feat. Alice)", for_artist=False
-        )
-        assert ftintitle.contains_feat(
-            "Live and Let Go [feat. Alice]", for_artist=False
-        )
-        assert ftintitle.contains_feat(
-            "Live and Let Go feat. Alice", for_artist=False
-        )
-        assert not ftintitle.contains_feat("Live and Let Go", for_artist=False)
-        assert not ftintitle.contains_feat("Come With Me", for_artist=False)
+        assert ftintitle.contains_feat("Live and Let Go (feat. Alice)")
```

Comment:
Ideally we'd keep non-varying data constant between the test cases: each test case should use **Alice** and **Bob** consistently so that it's clear which part is under test.

I see that `"Alice (feat. Bob)"` and `"Live and Let Go (feat. Alice)"` are testing the same thing, so the latter is redundant. This applies to all **Live and Let Go** test cases as their equivalents are already tested above.

Reply:
I have fixed this now